test/features/retro_idea_realtime_update_test.exs

```diff
@@ -1,6 +1,24 @@
 defmodule RetroIdeaRealtimeUpdateTest do
   use RemoteRetro.IntegrationCase, async: false
 
+  test "deleting an idea from a list", %{session: session_one, retro: retro} do
+    {:ok, session_two} = Wallaby.start_session()
+
+    retro_path = "/retros/" <> retro.id
+    session_one = authenticate(session_one) |> visit(retro_path)
+    session_two = authenticate(session_two) |> visit(retro_path)
+
+    idea = %{category: "sad", body: "user stories lack clear business value"}
+    submit_idea(session_two, idea)
+
```

Comment:
hey @Zanadar. poked around with this test for over an hour, and got false positives when disabling the push from the front end. Did some debugging, and found that the callback passed as a prop _does_ get invoked, but the function that that callback invokes isn't fired. 

Here's the output in Chrome:

<img width="450" alt="screen shot 2017-03-18 at 5 21 06 pm" src="https://cloud.githubusercontent.com/assets/4386645/24076836/d0766112-0c00-11e7-9396-6a3b68b4beb0.png">


And here's the output from the e2e test that deletes an idea:

![image](https://cloud.githubusercontent.com/assets/4386645/24076830/97c69440-0c00-11e7-8283-dec1f552ba20.png)
<img width="450" alt="screen shot 2017-03-18 at 5 21 06 pm" src="https://cloud.githubusercontent.com/assets/4386645/24076832/af585512-0c00-11e7-9cad-2fa1084dfc42.png">

I think the issue is that the `Room` component's `handleIdeaDeletion` method uses `.bind(this)`, and historically, bind and phantomjs haven't played well together. Gonna play around a bit more, but wanted to post this to see if it gave you any ideas.